redash/tasks/worker.py

```diff
@@ -22,10 +23,50 @@ def is_cancelled(self):
         return self.meta.get("cancelled", False)
 
 
+class StatsdRecordingQueue(BaseQueue):
+    """
+    RQ Queue Mixin that overrides `enqueue_call` to increment metrics via Statsd
+    """
+
+    def enqueue_call(self, func, args=None, kwargs=None, timeout=None,
+                     result_ttl=None, ttl=None, failure_ttl=None,
+                     description=None, depends_on=None, job_id=None,
+                     at_front=False, meta=None):
```

Comment:
You can do a catch all:

```suggestion
    def enqueue_call(self, *args, **kwargs):
```

This is more future proof, as you don't need to update when the parameters change.

Then the next line becomes:

```
job = super().enqueue_call(*args, **kwargs)
```

Reply:
ah yes python! thanks!